foia_hub/static/images/favicon/browserconfig.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+  <msapplication>
+    <tile>
+      <square70x70logo src="/static/images/favicon/mstile-70x70.png"/>

```

Comment:
I don't know enough about what's going on here to understand how this file is used. That said, we'll be using S3 to host our static assets. That means these paths won't work. 

@seanherron or @cmc333333 Have you encountered this before? Is there a standard way to tackle this?